plugins-builtin/src/main/scala/za/co/absa/enceladus/plugins/builtin/errorsender/params/ErrorSenderPluginParams.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.plugins.builtin.errorsender.params
+
+import java.time.Instant
+
+import za.co.absa.enceladus.plugins.builtin.errorsender.params.ErrorSenderPluginParams.ErrorSourceId
+
+case class ErrorSenderPluginParams(datasetName: String,
+                                   datasetVersion: Int,
+                                   reportDate: String,
+                                   reportVersion: Int,
+                                   outputPath: String,
+                                   sourceId: ErrorSourceId.Value,
+                                   sourceSystem: String,
+                                   runUrls: Option[String],
+                                   runId: Option[Int],
+                                   uniqueRunId: Option[String],
+                                   processingTimestamp: Instant
+                                ) {
+
+  def toMap: Map[String, String] = ErrorSenderPluginParams.toMap(this)
+}
+
+object ErrorSenderPluginParams {
+
+  object ErrorSourceId extends Enumeration {
+    val Standardization = Value("standardizaton")
+    val Conformance = Value("conformance")
+  }
+
+  object fieldNames {
```

Comment:
I would call it FieldName

Reply:
I completely agree (I had a `val` named like this before and the name got stuck) - fixed.